include/wx/setup_inc.h

```diff
@@ -757,7 +757,7 @@
 
 // notice that we can't use wxCHECK_VISUALC_VERSION() here as this file is
 // included from wx/platform.h before wxCHECK_VISUALC_VERSION() is defined
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
I think this test should be for `__MINGW64_TOOLCHAIN__`, AFAIK the old MinGW32 (of non-w64 variety) doesn't provide GDI+.